src/jit/codegencommon.cpp

```diff
@@ -8644,6 +8644,45 @@ void CodeGen::genFnEpilog(BasicBlock* block)
         unwindStarted = true;
     }
 
+#ifdef FEATURE_NGEN_RELOCS_OPTIMIZATIONS
```

Comment:
We try to avoid FEATURE ifdefs like these in the JIT. Can this be driven by what the VM tells JIT to do?

Reply:
@jkotas, yes, this would work without `#ifdef FEATURE_NGEN_RELOCS_OPTIMIZATIONS`. I'll remove it